docs/1-click-signup.mdx

```diff
@@ -0,0 +1,750 @@
+- **It allows you to keep the user in the same context.** If they start in a particular browser, by redirecting you can ensure they stay in that browser. (If they click the link, the mobile operating system may open it in a different browser.) If they start in a mobile app, you can open a webview within that app so they aren't taken away from it. (We recommend a full page webview, so it feels like part of the app.)
+
+
+#### Cons
+
+- **It's not ideal on non-mobile devices.** That's because ultimately the user has to verify on their mobile device — it's what allows us to pull verified data from the user's wireless phone provider. So if you redirect the user on a non-mobile device, our Wallet will display a QR code to direct them to their phone, which works but is not the best experience. Therefore, we recommend that if you redirect the user at all that you first check if they're on a mobile device and only do the redirect in that case.
+- **It's slightly more work to implement.** The [simplest implementation](#simplest-implementation) is not redirecting the user.
+
+
+### 3. Receive User Data {#receive-user-data}
+
+After you [call the 1-Click endpoint](#call-1-click-endpoint), you need to be set up to receive data the user consents to share with you.
+
+There are two options for how to do this:
+1. **[Push Option](#push-option) (recommended):** We _push_ the data to you, by hitting an endpoint you provide.
+
+OR
+
+2. **[Pull Option](#pull-option):** You _pull_ the data from us, by hitting an endpoint we provide.
+
+We recommend the push option because you are guaranteed to receive the data immediately, as soon as the user consents to share it. That results in 100% conversion.
+
+With the pull option, there's a time delay between when the user consents to share and when you are able to pull the data. That's because we have to redirect the user to you first. Conversion is generally somewhat less than 100% in this case.
+
+
+#### Push Option
+
+All you need to do is create a webhook called `POST /verified/credentials` according to our [OpenAPI spec](https://gist.github.com/VerifiedIncMachine/ef399f6b5ce6545eb4d7ec3e85d87dd2). This is where we will send verified user data when a user consents to share it.
+
+
+#### Pull Option
+
+Here's how this option works:
+1. We redirect to your `redirectUrl` and include a URL parameter called `sharedCredentialsUuid`.
+2. You call our [`GET /sharedCredentials`](/api-overview#get-shared-credentials) endpoint with the value of this parameter.
+
+
+#### Response
+
+The data you'll receive is the same, regardless of whether you use the push or pull option. 
+
+By default, you will receive a simplified response, so it's easy for you to take further action based on it (like creating a new account for the user). This simplified response only contains the user's verified data.
+
+But you can also choose to receive an encriched response, which contains much more metadata.
```

Comment:
How can they "choose to receive an enriched" response? It needs some details here... when we figure out how we want to handle it. If I recall, we were thinking a setting on brand.

Reply:
Good point. Let's discuss in technical implementation meeting tomorrow

Reply:
Maybe we:
- Set `enriched` it on the brand, where the default is false but it can be changed to true
- Add an optional attribute `enriched` to `GET /sharedCredentials`, where the default is false but it can be passed as true

Reply:
@leo1994 Is this possible yet? Can you clarify?